src/status_im/contexts/communities/actions/accounts_selection/view.cljs

```diff
@@ -5,81 +5,92 @@
-                                                       :t/member))]
-    [rn/safe-area-view {:style style/container}
-     [quo/page-nav
-      {:text-align          :left
-       :icon-name           :i/close
-       :on-press            #(rf/dispatch [:navigate-back])
-       :accessibility-label :back-button}]
-     [quo/page-top
-      {:title       (i18n/label :t/request-to-join)
-       :description :context-tag
-       :context-tag {:type           :community
-                     :size           24
-                     :community-logo (get-in images [:thumbnail :uri])
-                     :community-name name}}]
-     [gesture/scroll-view
-      [:<>
-       [quo/text
-        {:style               style/section-title
-         :accessibility-label :community-rules-title
-         :weight              :semi-bold
-         :size                :paragraph-1}
-        (i18n/label :t/address-to-share)]
-       [quo/category
-        {:list-type :settings
-         :data      [{:title             (i18n/label :t/join-as-a {:role highest-role-text})
-                      :on-press          #(rf/dispatch [:open-modal :addresses-for-permissions
-                                                        {:community-id id}])
-                      :description       :text
-                      :action            :arrow
-                      :label             :preview
-                      :label-props       {:type :accounts
-                                          :data selected-accounts}
-                      :description-props {:text (i18n/label :t/all-addresses)}}
-                     {:title             (i18n/label :t/for-airdrops)
-                      :on-press          #(rf/dispatch [:open-modal :airdrop-addresses
-                                                        {:community-id id}])
-                      :description       :text
-                      :action            :arrow
-                      :label             :preview
-                      :label-props       {:type :accounts
-                                          :data [airdrop-account]}
-                      :description-props {:text (:name airdrop-account)}}]}]
-       [quo/text
-        {:style               style/section-title
-         :accessibility-label :community-rules-title
-         :weight              :semi-bold
-         :size                :paragraph-1}
-        (i18n/label :t/community-rules)]
-       [community-rules/view id]]]
-     [rn/view {:style (style/bottom-actions)}
-      [quo/slide-button
-       {:size                :size-48
-        :track-text          (i18n/label :t/slide-to-request-to-join)
-        :track-icon          :i/face-id
-        :customization-color color
-        :on-complete         #(join-community-and-navigate-back id)}]]]))
+  (let [{id :community-id}             (rf/sub [:get-screen-params])
+        show-addresses-for-permissions (fn []
```

Comment:
FYI: This is a performance optimization pattern we can apply more often. The community ID or most IDs for that matter will never change, so we can extract the community ID to a `let` binding on the mount phase. Along with that, we can then extract anonymous functions that are re-created on every render that need to close over the community ID, therefore allowing Reagent to detect that the arguments passed to hiccup components are the same when it compares args for the current and new render.